src/minErr.js

```diff
@@ -82,6 +82,11 @@ function isValidObjectMaxDepth(maxDepth) {
 
 function minErr(module, ErrorConstructor) {
   ErrorConstructor = ErrorConstructor || Error;
+
+  var url = 'https://code.angularjs.org/"NG_VERSION_FULL"/docs/errors/';
```

Comment:
Why not use `https://errors.angularjs.org/...`?

Reply:
errors. ist just a redirect. Why not cut out the middle men?

Reply:
It is shorter and cleaner :grin:

I prefer the `errors.angularjs.org` URLs, but don't feel too strongly about it.
But I do think it is a good idea to have the same URLs in non-minified and minified versions, so if we change it here, I would like to see it changes in `ng-closure-runner` too 😇